### PR TITLE
conditional logic fix

### LINKED ISSUE #57: SPA example code is invalid
This part of authRedirect.js seems like a mistake:

https://github.com/Azure-Samples/active-directory-b2c-javascript-msal-singlepageapp/blob/ef99a862a05445b55aaa76425a3cb6b4b01920b4/JavaScriptSPA/authRedirect.js#L22-L28

Maybe something was missed in the recent example rewrite?

### PR DESCRIPTION
**Pull request by derisen**
This PR corrects a mistake with conditional logic in `authRedirect.js`, where

` if (accessToken === null || accessToken === undefined)`

should have been 

` if (accessToken !== null || accessToken !== undefined)` or just ` if (accessToken)`

Nothing here should affect documentation as we do not discuss `authRedirect.js` in them @mmacy

### CODE REVIEW

File: JavaScriptSPA/authConfig.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
-// Config object to be passed to Msal on creation.
-// For a full list of msal.js configuration parameters, 
-// visit https://azuread.github.io/microsoft-authentication-library-for-js/docs/msal/modules/_configuration_.html
+/**
+ * Config object to be passed to Msal on creation.
```
Comment (mmacy) [inline JavaScriptSPA/authConfig.js]:
```suggestion
 * Config object to be passed to MSAL on creation.
```

**Review by mmacy — APPROVED** [review]
LGTM, thanks @derisen!

File: JavaScriptSPA/authRedirect.js [hunk 2]
```diff
@@ -12,22 +12,22 @@ function authRedirectCallBack(error, response) {
...
       myMSALObj.getAccount();
       getTokenRedirect(tokenRequest);
     } else if (response.tokenType === "access_token") {
-        console.log('access_token acquired at: ' + new Date().toString());
+        console.log("access_token acquired at: " + new Date().toString());
         accessToken = response.accessToken;
-        logMessage("Request made to Web API:")
-        if (accessToken === null || accessToken === undefined) {
+        logMessage("Request made to Web API:");
+        if (accessToken) {
```
Comment (jasonnutter) [inline JavaScriptSPA/authRedirect.js:22]: Before this was checking if `accessToken` was falsey, now it appears to be checking if it is truthy, that intended?

Comment (derisen) [reply JavaScriptSPA/authRedirect.js:22]: that's right, the intention was to check for accessToken before calling the api, or acquire it if it is not (down below). The 'if' statement for the two was mixed up..

**Review by jasonnutter — APPROVED** [review]

**Pull request merged**